examples/stable/test/property/ic_api/msg_arg_data/test/tests.ts

```diff
@@ -1,4 +1,4 @@
-globalThis._azleExperimental = true;
+import 'azle/experimental/_internal/test/set_experimental';
```

Comment:
Let's get rid of all of the `globalThis._azleExperimental = true` things.

Reply:
Apparently I already had gotten rid of all of them